docs/walkthrough/integrate-with-jina.md

```diff
@@ -59,6 +59,25 @@ Text of the returned document: some text to encode
 Shape of the embedding: (768,)
 ```
 
+### Encoding a List
+Data that is stored in a regular list can be embedded in the same way you would a [DocumentArray](https://docarray.jina.ai/). Since the modality of your input data can be inferred from the model being used, there is not need to provide any additional information besides the content you want to encode:
+
+```python
+from docarray import DocumentArray, Document
+import finetuner
+
+model = finetuner.get_model('/path/to/YOUR-MODEL.zip')
+
+lst = ['some text to encode']
+
+da = finetuner.encode(model=model, data=lst)
+
+for doc in da:
+    print(f'Text of the returned document: {doc.text}')
+    print(f'Shape of the embedding: {doc.embedding.shape}')
```

Comment:
if user provide a list of object, we should not convert it as a da, but an numpy ndarray of embeddings.

Reply:
That makes sense, changed now